src/components/CardElementText.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "styled-components";
+
+const Wrapper = styled.div`
+  display: flex;
+
+  flex-direction: column;
+  height: 150px;
+  width: 60%;
+  justify-content: flex-end;
+  margin-left: 10px;
+`;
+
+const StudioName = styled.h2`
+  color: rgb(110, 94, 94);
+  font-family: Arial, Helvetica, sans-serif;
+`;
+
+const MonthlyPrice = styled.span`
+  color: rgb(110, 94, 94);
+  font-family: Arial, Helvetica, sans-serif;
+`;
+const Adress = styled.div`
+  font-family: Arial, Helvetica, sans-serif;
+  color: rgb(110, 94, 94);
+`;
+
+export default function CardElementText({ name, price, adress }) {
```

Comment:
Maybe it would be easier to create objects that represent a studio and then access the information from that object. This will make it easier to implement the DataBase later on.

Reply:
Si  (Shaming Monkey Smily)